common/Readme

```diff
@@ -1 +1 @@
-Code shared between client and server
+Code shared between client and server.
```

Comment:
While your at it, this line is actually inaccurate. It should read:
General purpose utility classes. Not Pravega specific.